### DISCUSSION TITLE
Pokecube AIO

### DISCUSSION THREAD

**Discussion by supermassimo** [DynamicTreesTeam/DynamicTrees#911] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/pokecube-aoi
+ 2 dimensions
+ 18 trees

**Comment by WenXin20** [comment]
- 12 berries that grow into trees when planted on farmland & found in various biomes, one of which can only in The End
- 5 trees found in Ultraspace
- 1 tree found in the Distorted World

![image](https://github.com/user-attachments/assets/f85f0a9e-12e1-4916-8c0f-be3aadb5781c)
![image](https://github.com/user-attachments/assets/4d450cec-a021-4d8b-a327-b8bb71de93e3)
![image](https://github.com/user-attachments/assets/d28b2e23-871e-4891-a9e8-e62617378ccc)
